container/fs/fs.go

```diff
@@ -52,11 +54,12 @@ type RealFsInfo struct {
 	labels map[string]string
 }
 
-var collector DiskUsageCollector
+//var collector DiskUsageCollector
 
 type DiskUsageCollector struct {
-	Mut       *sync.Mutex
-	DiskUsage map[string]uint64
+	Mut         *sync.Mutex
```

Comment:
Consider re-use here the comment from [client.go#L5](https://github.com/marcin-krolik/snap-plugin-collector-docker/blob/feat/refactor/container/client.go#L5) about DiskUsageCollector